test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -503,7 +503,9 @@ public static HapiSpecOperation updateLargeFile(
 			boolean signOnlyWithPayer,
 			OptionalLong tinyBarsToOffer
 	) {
-		return updateLargeFile(payer, fileName, byteString, signOnlyWithPayer, tinyBarsToOffer, op -> {}, op -> {});
+		return updateLargeFile(payer, fileName, byteString, signOnlyWithPayer, tinyBarsToOffer, op -> {
+		}, op -> {
+		});
```

Comment:
nit: Not sure if this code snippet is formatted by Intellij or not. It would be better to have it like if possible:
```
return updateLargeFile(payer, fileName, byteString, signOnlyWithPayer, tinyBarsToOffer, 
                 op -> {}, op -> {});
```

Reply:
I think it's the IntelliJ on Neeha's computer

Reply:
@ljianghedera  @qnswirlds  Yes, my IntelliJ formatted the line. Fixed it. Thank you!